v2/yarpcprotobuf/protoc-gen-yarpc-go/internal/tests/gen/proto/src/keyvalue/key_value.pb.yarpc.go.fixture

```diff
@@ -194,7 +220,8 @@ type FxStoreYARPCServerResult struct {
 func NewFxStoreYARPCServer() interface{} {
 	return func(p FxStoreYARPCServerParams) FxStoreYARPCServerResult {
 		return FxStoreYARPCServerResult{
-			Procedures: BuildStoreYARPCProcedures(p.Server),
+			UnaryProcedures: BuildUnaryStoreYARPCProcedures(p.Server),
```

Comment:
There’s a funny thing about english adjective order. https://dictionary.cambridge.org/us/grammar/british-grammar/about-adjectives-and-adverbs/adjectives-order

I propose that Build Unary Store YARPC Procedure probably should be Build Unary YARPC Store Procedures, but I can’t speak for how awful it would be to make that happen.

Reply:
TIL about adjective ordering.

But yes, the templating for that to happen is a bit complicated.